core/src/main/java/org/stellar/anchor/sep6/Sep6TransactionUtils.java

```diff
@@ -0,0 +1,74 @@
+package org.stellar.anchor.sep6;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.stellar.anchor.api.shared.RefundPayment;
+import org.stellar.anchor.api.shared.Refunds;
+
+public class Sep6TransactionUtils {
+
+  /**
+   * Converts a SEP-6 database transaction object to a SEP-6 API transaction object.
+   *
+   * @param txn the SEP-6 database transaction object
+   * @return the SEP-6 API transaction object
+   */
+  public static org.stellar.anchor.api.sep.sep6.Sep6Transaction fromTxn(Sep6Transaction txn) {
```

Comment:
```suggestion
  public static Sep6Transaction fromTxn(Sep6Transaction txn) {
```

Reply:
Unfortunately, we have two classes with the same name.

Reply:
Ah, it's because SEP-24/31 call the platform transaction class something else. https://github.com/stellar/java-stellar-anchor-sdk/blob/625e469bb7c55ffa77d7cf0906b1b926da967fa0/core/src/main/java/org/stellar/anchor/sep24/Sep24Helper.java#L81.

I will rename the class too.